modules/glossary-of-common-terms-for-rhoai.adoc

```diff
@@ -0,0 +1,85 @@
+:_module-type: REFERENCE
+
+[id='glossary-of-common-terms-for-{productname-short}_{context}']
+= Glossary of common terms for {productname-short}
+
+This glossary defines common terms for {productname-long}.
+
+accelerator:: In high-performance computing, a specialized circuit that is used to take some of the computational load from the CPU, increasing the efficiency of the system. For example, in deep learning, GPU-accelerated computing is often employed to offload part of the compute workload to a GPU while the main application runs off the CPU 
+//Reference: IBM glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x2048370
+
+artificial intelligence (AI):: The capability to acquire, process, create and apply knowledge in the form of a model to make predictions, recommendations or decisions.
+//Reference: https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x3448902
+
+bias detection:: The process of calculating fairness metrics to detect when AI models are delivering unfair outcomes based on certain attributes.
+//Reference:: IBM glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x9721361
+
+cluster storage:: A persistent volume that retains the files and data within a workbench. A workbench has access to one or more cluster storage instances.
+
+custom resource (CR):: A resource implemented through the Kubernetes CustomResourceDefinition API. A custom resource is distinct from the built-in Kubernetes resources, such as the pod and service resources. Every CR is part of an API group.
+//Reference: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource
+
+custom resource definition (CRD):: In Red Hat OpenShift, a custom resource definition (CRD) defines a new, unique object `Kind` in the cluster and lets the Kubernetes API server handle its entire lifecycle.
+// Reference:: https://redhat-documentation.github.io/supplementary-style-guide/#custom-resource-definition
+
+data connection:: The configuration parameters required to connect to a data source, such as an S3 object bucket.
+
+data science pipelines:: A portable machine learning (ML) workflow to develop and deploy data science models.
+
+data science project:: An isolated environment for developing, training and serving machine learning models.
```

Comment:
attempt at alternative definition: openshift namespace used for openshift AI

data science project really is just an openshift project

Reply:
Despite some customers being unhappy with that, it is how it is

* https://github.com/red-hat-data-services/ods-ci/issues/2279